### ISSUE TITLE
Documentation Update / Howto?

### ISSUE THREAD

**Issue by djl197** [reviewboard/ReviewBot#17]
I have the new version of the reviewbot installed and its finding the tools on my remote 'bots'
However the configuration of these is now rather different and I am slightly confused.

Is there any chance of a quick howto setup the ReviewBots?
I am thinking about Tool config and then Automatic/Manual Run groups as I dont really know what I am doing :-)

Cheers
Dan

**Comment by anselina** [comment]
Hi Dan,

You'll want to use the release-0.2.x branch (which will have the same admin config options you're used to), instead of master. master is still under development -- there are a few things we need to do before it can be released. We'll be sure to update the readme with that release. :)

Anselina

**Comment by djl197** [comment]
Hi AnselinaMany thanks for getting back to me,  I appreciate that its a work in progress but what there is seems to almost be working for me.I have added profiles for the tools, then set up automatic run groups based on file extensions and given them the profiles.It seems to be posting the feedback to the reviews.So the only thing that I cant quite get working is passing the parameters to the tools them selves.Does this work?If it does not then probably will switch back to 0.2, however if you have an example JSON config (for the tool settings field) for jshint then I am happy to be an alpha tester and give feedback.I would much rather use this as it looks a huge improvement - hope you can helpDan  

     On Thursday, 8 January 2015, 19:04, Anselina Chia <notifications@github.com> wrote:
   

 Closed #17.—

**Comment by anselina** [comment]
Glad to hear that master seems to work properly! The Tool Settings admin UI is actually one of last things we need to fix, as you noticed. Either way, it'd be great to have more testing done. I don't have an example handy, but if I remember correctly, something like this should work for the JSHint tool: 
```
{ 
  "verbose": true  // or false
  "extra_ext_checks": ""  // or ".json,.html"
  "extract_js_from_html": "never"   // or "auto" or "always"
  "config": // paste contents of your .jshintrc file here
}
```
Just a note that some of the new/fun features (manual execution of tools, viewing tool execution statuses) are accessible through the API, but won't actually show up anywhere. Tool "profiles" and automatic execution via AutomaticRunGroups should both work fine though.
